### PR TITLE
we changed the limit change value from .5 to .01

### PR DESCRIPTION
**Pull request by jerestet37**
we changed the limit change value from .5 to .01

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -132,25 +132,25 @@ public void teleopPeriodic() {
...
-    List<DeviceOutputCommand> shooterCommands = shooterSubsystem.run(model.shooterModel.orElse(new ShooterSubsystemModel(ShooterSubsystemModel.ShooterState.STOPPED)));
-    List<DeviceOutputCommand> conveyorCommands = conveyorSubsystem.run(model.conveyorModel.orElse(
-        new ConveyorSystemModel(
-          ConveyorSystemModel.IntakeState.STOPPED,
-          ConveyorSystemModel.IntakePosition.UP,
-          ConveyorSystemModel.ShooterBlockState.CLOSE
-        )
-    ));
-    List<DeviceOutputCommand> hangerCommands = hangerSubystem.run(model.hangerModel.orElse(new HangerSystemModel(HangerSystemModel.HangerState.STOPPED)));
+    // List<DeviceOutputCommand> shooterCommands = shooterSubsystem.run(model.shooterModel.orElse(new ShooterSubsystemModel(ShooterSubsystemModel.ShooterState.STOPPED)));
```
Comment (tylerallen40) [inline src/main/java/frc/robot/Robot.java]: Lets uncomment these lines before merging, it is not correct and also causing test failures

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -132,25 +132,25 @@ public void teleopPeriodic() {
...
 
 
     hardwareInterface.run(
         //Don't worry about this code, I know it is confusing, but it does make sense
         Stream.of(
-            hangerCommands,
-            driveMotorCommands,
-            shooterCommands, 
-            conveyorCommands)
+            // hangerCommands,
```
Comment (tylerallen40) [inline src/main/java/frc/robot/Robot.java]: same as above

File: src/main/java/frc/robot/controllers/TankDrive.java [hunk 3]
```diff
@@ -37,8 +37,8 @@ public double limitChange(double target, double current, double maxDiff) {
 
   @Override
   public RobotModel run(HashMap<String, InputContainer<?>> inputMap) {
-    this.currentValueLeft = this.limitChange((double)inputMap.get(this.leftJoystickName).getValue(), this.currentValueLeft, 0.4);
-    this.currentValueRight = this.limitChange((double)inputMap.get(this.rightJoystickName).getValue(), this.currentValueRight, 0.4);
+    this.currentValueLeft = this.limitChange((double)inputMap.get(this.leftJoystickName).getValue(), this.currentValueLeft, 0.01);
```
Comment (tylerallen40) [inline src/main/java/frc/robot/controllers/TankDrive.java:40]: This could use some more fine tuning possibly, but it is fine for now

**Review by tylerallen40 — CHANGES_REQUESTED** [review]

**Review by tylerallen40 — APPROVED** [review]

**Review by jerestet37 — COMMENTED** [review]
looks good

**Pull request merged**